hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -190,15 +190,21 @@ const COLUMNS = [
     render: (text: DatanodeOpState) => renderDatanodeOpState(text),
     sorter: (a: IDatanode, b: IDatanode) => a.opState.localeCompare(b.opState)
   },
-
   {
     title: 'Uuid',
     dataIndex: 'uuid',
     key: 'uuid',
     isVisible: true,
     isSearchable: true,
     sorter: (a: IDatanode, b: IDatanode) => a.uuid.localeCompare(b.uuid),
-    defaultSortOrder: 'ascend' as const
+    defaultSortOrder: 'ascend' as const,
+    render: (uuid: IDatanode, record: IDatanode) => {
+      return (
+        //1. Compare Decommission Api's UUID with all UUID in table and show Decommission Summary
+        (decommissionUids && decommissionUids.includes(record.uuid) && record.opState !== 'DECOMMISSIONED') ? 
```

Comment:
Minor nit for UUID spelling
```suggestion
        (decommissionUuids && decommissionUuids.includes(record.uuid) && record.opState !== 'DECOMMISSIONED') ? 
```

Reply:
Done changes in latest 2 commits